style.css

```diff
@@ -0,0 +1,1100 @@
+}
+
+ul#elements li a {
+  white-space: nowrap;
+  font-weight: 600;
+  font-size: 8.53vw;
+  line-height: 11.73vw;
+  color: white;
+  text-decoration: none;
+}
+
+div#background {
+  display: none;
+  height: 102vh;
+  width: 102vw;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: rgba(219, 80, 55, 0.8);
+  -webkit-backdrop-filter: blur(10px);
+  backdrop-filter: blur(10px);
+  z-index: 1;
+}
+
+section {
+  opacity: 0.95;
+}
+
+section nav #mobileButton {
+  background-image: url(images/menu.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 2.5rem;
+  height: 2.5rem;
+  margin-left: 4%;
+  margin-top: 0;
+  margin-bottom: 4%;
+}
+
+section nav ul {
+  visibility: hidden;
+  position: absolute;
+}
+
+section.presentation {
+  background-color: #f3f0f0ee;
+  opacity: 0.95;
+}
+
+section.presentation section.allpresentation div {
+  margin: 0 3vw;
+}
+
+section.presentation section.allpresentation div h1 {
+  margin: 0 0 5vw;
+  color: #ec5242;
+  font-weight: 900;
+  font-size: 3rem;
+  mix-blend-mode: multiply;
+}
+
+.smalltitle {
+  font-size: 1.5rem;
+  font-weight: 300;
+}
+
+section.presentation section.allpresentation div div.info {
+  margin: 0;
+  border: 8px white solid;
+}
+
+section.final div p {
+  padding: 2%;
+  font-size: 0.7rem;
+  color: #9f9ea2;
+  font-weight: 700;
+  max-width: 30ch;
+  visibility: hidden;
+}
+
+section.presentation section.allpresentation div h2 {
+  text-align: center;
+  font-size: 0.9rem;
+  color: #5a5a5a;
+  font-weight: 700;
+  padding-top: 20px;
+  margin-left: -45%;
+}
+
+section#containercards section.cards section.individualcard p {
+  position: relative;
+  font-weight: 600;
+  color: #fff;
+  width: 50%;
+  font-size: 1.2rem;
+  margin-left: 45%;
+  margin-top: -23%;
+}
+
+img.simbols {
+  margin-top: 5%;
+}
+
+.subdescription {
+  color: #5a5a5a;
+  position: absolute;
+  text-align: center;
+  padding-top: 1%;
+  padding-bottom: -20%;
+  font-weight: 800;
+  font-size: 0.88rem;
+  width: 100%;
+  margin-right: 5%;
+}
+
+section#containercards {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #272a31;
+  background-image: url(images/background.png);
+  background-size: 8vw;
+  background-repeat: repeat;
+  margin-top: 25%;
+}
+
+section#containercards h3 {
+  font-size: 1.5rem;
+  font-weight: 400;
+  padding-top: 9vw;
+  color: white;
+  margin: 0;
+}
+
+.amazon {
+  margin-top: 10px;
+}
+
+.google {
+  margin-top: 30px;
+}
+
+section#containercards hr {
+  border-top: 0;
+  border: 0.2vw solid #ec5242;
+  width: 10vw;
+  margin: 3vw 0 8vw;
+}
+
+section#containercards section.cards {
+  display: flex;
+  flex-direction: column;
+  gap: 2vw;
+}
+
+section#containercards section.cards section.individualcard {
+  background-color: rgba(161, 159, 159, 0.3);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+  height: 3%;
+  width: -5%;
+}
```

Comment:
- [ ] Well done 👏🏻 , you did an awesome job by implementing the changes as per the previous code reviewer's request 👍🏻 but it seems like your design is still not matching the **Behance** design, your previous code reviewer asked for adding the icon for the `Main Program` section but it seems like your icon are not showing up properly on the wider screen sizes. Please fix the issue by correctly using the CSS styles to achieve a better result. I have attached an image below which will help you to understand the problem.


### Video
[Screencast from 03-08-23 12:16:42 AM IST.webm](https://github.com/maximoortelli/Capstone-Proyect/assets/66476329/669c69ae-f668-48da-aa3d-e355f69b5c0a)

Reply:
I have done it, thank you very much for correcting me and sorry for the inconvenience. I have already done everything requested, I hope you can approve me in this last Pull Request, otherwise I will be left out.